Menu/ContextMenu: add Scrolling demo descriptions (#27315)
DevExpress/DevExtreme: apps/demos/Demos/ContextMenu/Scrolling/description.md

> The ContextMenu component supports submenu item scrolling. If the combined item length exceeds the screen size, a scrollbar appears, and the submenu height is restricted.

Reviewer: Judging by information in the next file, you can also limit the height manually. The limit doesn't have to be the screen size. So I'm adding that part. And then it's clear that the submenu is restricted (since I'm already mentioning height restrictions). 

```suggestion
The ContextMenu component supports submenu item scrolling. If the combined item height exceeds the screen size or a pre-defined height limit, a scrollbar appears.
```

> The ContextMenu component supports submenu item scrolling. If the combined item length exceeds the screen size, a scrollbar appears, and the submenu height is restricted.

Author: ```suggestion
The ContextMenu component supports submenu item scrolling. If the combined item height exceeds the screen size, a scrollbar appears.
```
